addons/website/views/snippets/s_popup.xml

```diff
@@ -3,25 +3,17 @@
 
 <template id="s_popup" name="Popup">
     <div class="s_popup o_snippet_invisible">
-        <div class='modal s_popup_main popup_backdrop s_popup_center o_one_page bg-black-25 '
-             data-show-after="5000" data-display="afterDelay" data-consents-duration="7">
-            <div class='modal-dialog modal-dialog-centered s_popup_frame modal-md'>
-                <div class='oe_structure modal-content s_popup_content oe_img_bg oe_custom_bg' style="background-color: #FFFFFF">
-                    <div class='js_close_popup s_popup_close o_we_no_overlay o_not_editable'>&amp;times;</div>
-                    <section class="s_banner parallax pt96 pb96" data-snippet="s_banner">
-                        <span class="s_parallax_bg oe_img_bg oe_custom_bg" style="background-image: url('/web/image/website.s_banner_default_image');"></span>
+        <div class="modal fade s_popup_middle bg-black-50" data-show-after="5000" data-display="afterDelay" data-consents-duration="7" tabindex="-1" role="dialog">
+            <div class="modal-dialog modal-dialog-centered">
+                <div class="modal-content oe_structure bg-white">
```

Comment:
Is bg-white useful? Meh... to review with the new colorsystem, there are too many color level here :/